xonsh/ptk_shell/completer.py

```diff
@@ -98,11 +98,17 @@ def get_completions(self, document, complete_event):
         for comp in completions:
             # do not display quote
             if isinstance(comp, RichCompletion):
+                desc = (
+                    comp.description.replace(os.linesep, " ")
```

Comment:
Maybe `. ` (dot newline) so the sentences appear split?

Reply:
putting dot would mean certain meaning. then we may want to title the new start ...

Reply:
we can add some special characters like the return character. is that supported in most of the terminals? or we can have that in a setting ?

Reply:
I don't think this is a big deal
Just decide on something that looks nice

Reply:
space is fine for now